plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/roi_align.cpp

```diff
@@ -2,62 +2,45 @@
+    ::testing::ValuesIn(std::vector<float>{1, 0.625}),    //
+    ::testing::Values(2),                                 //
+    ::testing::Values("avg"),                             //
+    ::testing::Values(InferenceEngine::Precision::FP32),  //
+    ::testing::Values(CommonTestUtils::DEVICE_PLAIDML)    //
 );
 
-INSTANTIATE_TEST_CASE_P(ROIAlign, ROIAlignLayerTest,
-                        ::testing::Combine(roiAlignParams,                                       //
-                                           ::testing::ValuesIn(netPrecisions),                   //
-                                           ::testing::Values(CommonTestUtils::DEVICE_PLAIDML)),  //
-                        ROIAlignLayerTest::getTestCaseName);
-
-const auto smokeRoiAlignParams = ::testing::Combine(::testing::Values(inputShapes[0]),  //
-                                                    ::testing::Values(rois),            //
-                                                    ::testing::Values(batchIndices),    //
-                                                    ::testing::Values(2),               //
-                                                    ::testing::Values(3),               //
-                                                    ::testing::Values(3),               //
-                                                    ::testing::Values(0),               //
-                                                    ::testing::Values(20.0f),           //
-                                                    ::testing::Values("max")            //
+// INSTANTIATE_TEST_CASE_P(smoke_TestsROIAlign_average, ROIAlignLayerTest, ROIAlignCases_average,
+//                        ROIAlignLayerTest::getTestCaseName);
+
+const auto ROIAlignCases_max = ::testing::Combine(  //
+    ::testing::ValuesIn(std::vector<std::vector<size_t>>{
+        {2, 8, 20, 20},
+        {2, 1, 20, 20},
+    }),                                                   //
+    ::testing::Values(std::vector<size_t>{2, 4}),         //
+    ::testing::Values(2),                                 //
+    ::testing::Values(2),                                 //
+    ::testing::ValuesIn(std::vector<float>{1, 0.625}),    //
+    ::testing::Values(2),                                 //
+    ::testing::Values("max"),                             //
+    ::testing::Values(InferenceEngine::Precision::FP32),  //
+    ::testing::Values(CommonTestUtils::DEVICE_PLAIDML)    //
 );
 
-INSTANTIATE_TEST_CASE_P(smoke, ROIAlignLayerTest,
-                        ::testing::Combine(smokeRoiAlignParams,                                  //
-                                           ::testing::ValuesIn(netPrecisions),                   //
-                                           ::testing::Values(CommonTestUtils::DEVICE_PLAIDML)),  //
-                        ROIAlignLayerTest::getTestCaseName);
-}  // namespace
+// INSTANTIATE_TEST_CASE_P(smoke_TestsROIAlign_max, ROIAlignLayerTest, ROIAlignCases_max,
```

Comment:
Is this the part you're still working on? (and the commented out test cases in RoI Pooling as well)

Reply:
Yes, we are still working on updating Roi Pooling and RoI Align.